### ISSUE TITLE
FR: [RoadChoice]: Pardot Forms Block

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-roadchoice-com#68]
**Description**
Available pardot forms:
- https://roadchoice.com/subscribe
- https://roadchoice.com/contact-us
- https://roadchoice.com/bus

**Acceptance criteria**
- The pardot form block is available on RoadChoice, following the brand's identity in terms of styling
- If cookies are not accepted, the form is not loaded on the page 

**Open points**
- Q: What do we show instead, when the user has not accepted the cookies
- A: @shirin27 Check what is already done for Eloqua, and also align with the Config OneTrust approach in the corresponding ticket
- For captcha/spam bot protection, @manuel-vara to check with Ryan if we can use honeypot captcha

**Comment by shirin27** [comment]
To be estimated after alignment with Ryan on spam bot protection and integration.

**Comment by manuel-vara** [comment]
Q: What do we show instead, when the user has not accepted the cookies?
- Tracking will be disabled from Pardot. There's a feature to disable it and only use it to gather form data.
A: @shirin27 Check what is already done for Eloqua, and also align with the Config OneTrust approach in the corresponding ticket.

For captcha/spam bot protection, @manuel-vara to check with Ryan if we can use honeypot captcha.
Using IFrames, Pardot provides honeypot configuration and ReCaptcha, both are foreseen to be enabled form Pardot's side.

**Comment by manuel-vara** [comment]
We will be using IFrame Block with fixed height provided by Pardot form configuration. Advanced cases where dynamic height based on user's input/screen or other data communications between parent page and Pardot form, would be treated separately in another ticket.

**Comment by girishdigrajkar** [comment]
Validated and 'Subscribe' & 'Contact-us' forms works as expected.

'Bus' form is pending validation and is blocked by #88. Post resolution of #88 testing will be resumed for this ticket.
